roles/forklift_versions/molecule/default/INSTALL.rst

```diff
@@ -0,0 +1,22 @@
+*******
+Docker driver installation guide
+*******
+
+Requirements
+============
+
+* Docker Engine
```

Comment:
Is it possible to run with podman instead of docker?

Reply:
yeah, or no container at all, like I actually do here, so removed that file.